web/src/tippyjs.ts

```diff
@@ -193,6 +193,15 @@ export function initialize(): void {
         },
     });
 
+    tippy.delegate("body", {
+        target: ".unsubscribed_icon",
+        placement: "right",
+        appendTo: () => document.body,
+        onHidden(instance) {
+            instance.destroy();
+        },
+    });
```

Comment:
I don't think we even need this. Let's use  `tippy-zulip-tooltip` and squash both the commit.

Reply:
tippy-zulip-tooltip-unsubscribe ? As tippy-zulip-tooltip is for more elements.